src/main/scala/org/camunda/feel/impl/interpreter/BuiltinFunctions.scala

```diff
@@ -1034,6 +1033,13 @@ object BuiltinFunctions extends FunctionProvider {
       case e => error(e)
     })
 
+  def decimalFunction3 =
+    ValFunction(List("n", "scale", "mode"), _ match {
+      case List(ValNumber(n), ValNumber(scale), ValString(mode)) =>
+        ValNumber(n.setScale(scale.intValue, RoundingMode.withName(mode)))
```

Comment:
To avoid exceptions, we should check if the mode is a valid `RoundingMode` (in a case-insensitive way). If not then we should return a `ValError` that also lists all valid rounding modes.